datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/refresh.py

```diff
@@ -9,17 +9,24 @@
 import pywintypes
 import win32pdh
 
+try:
+    import datadog_agent
+except ImportError:
+    datadog_agent = None
+
 from ....utils.time import get_precise_time
 
 
 class WindowsPerformanceObjectRefresher(threading.Thread):
-    INTERVAL = 60
-
     def __init__(self):
         name = self.__class__.__name__
         super(WindowsPerformanceObjectRefresher, self).__init__(name=name)
 
+        self.interval = 60
+        if datadog_agent and datadog_agent.get_config('windows_counter_refresh_interval'):
```

Comment:
Checking for truthyness prevents the ability to set to zero

Reply:
Theoretically a good find but would not this prevent refreshes thread in a first place https://github.com/DataDog/integrations-core/pull/12665/files#diff-77f24038c0b7c7947277e93e51cf208efa6f8ddfbed47213c566533a6b0ba4f7R46-R47?

Reply:
Thanks for that. I edited it just to keep things more concise.